monitoring/mock_uss/config.py

```diff
@@ -40,3 +40,6 @@ class Config(object):
     DSS_URL = os.environ.get(ENV_KEY_DSS, None)
     BEHAVIOR_LOCALITY = Locality(os.environ.get(ENV_KEY_BEHAVIOR_LOCALITY, "CHE"))
     CODE_VERSION = os.environ.get(KEY_CODE_VERSION, "Unknown")
+    CERT_PATH = "/var/test-certs"
```

Comment:
This class is where we define configurable variables; we wouldn't want to put a constant here.

I think it probably would make sense to specify a CERT_BASE_PATH as a configurable variable like this, and this would be a reasonable default value.  However, to make it configurable, we'd want to follow the design pattern for the other configurable variables which, to accommodate Flask's somewhat wonky configuration definition and ensure traceability, includes:

1. Define environment variable of the configuration variable (probably `ENV_KEY_CERT_BASE_PATH = "{}_CERT_BASE_PATH".format(ENV_KEY_PREFIX)` to match previous style)
2. Define a constant for the config key name to avoid magic strings in different files (probably `KEY_CERT_BASE_URL = "CERT_BASE_URL"`)
3. Add the configuration variable the config object, loading the value from the environment variable (probably `CERT_BASE_URL = os.environ.get(ENV_KEY_CERT_BASE_URL, "/var/test-certs")`)

At that point, the value can be used at runtime with [`webapp.config.get(KEY_CERT_BASE_URL)`](https://github.com/interuss/dss/blob/17c4f01bd787014bcba0cc6942131192398834ae/monitoring/mock_uss/ridsp/routes_injection.py#L48).